### PR TITLE
chore: add colors to the tree

### PR DESCRIPTION
**Pull request by L-Joli**
- add colors to the tree
green
red

### CODE REVIEW

File: src/ui/targetingTree.ts [hunk 1]
```diff
@@ -156,4 +165,16 @@ const insertScheduleTree = (rootTree: Tree, rollout?: Rollout) => {
 
         rootTree.insert('gradual rollout', scheduleTree)
     }
+}
+
+const coloredTitle = (title: string) => {
+    return chalk.hex('#93E3FE')(title)
```
Comment (laurawarr) [inline src/ui/targetingTree.ts:171]: it seems like it'd be worthwhile defining these hex codes as constants, maybe in the UI folder? then we can easily reuse these colours

File: src/ui/constants/colors.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const COLORS = {
+    lightBlue: '#93E3FE',
+    paleGreen: '#B0DC8B',
+    paleYellow: '#FDF894',
```
Comment (laurawarr) [inline src/ui/constants/colors.ts:4]: just a naming nitpick, but why not stick with either "pale" or "light"?

**Review by laurawarr — APPROVED** [review]
one comment, but otherwise looks good 👍

File: src/ui/constants/colors.ts (hunk 2 above)
Comment (L-Joli) [reply src/ui/constants/colors.ts:4]: 😂 I'm usually bad at choosing names, so I followed chatGPT suggestions for the color names, but you're right, let's stick to 'light'🤣

**Pull request merged**